### DISCUSSION TITLE
conferences

### DISCUSSION THREAD

**Discussion by icsabai** [csabaiBio/elte_ml_journal_club#94] [Announcements]
__AAAI Spring Symposium on Combining  Artificial Intelligence and Machine Learning with Physics Sciences (MLPS)__

March 22-24, 2021 @Stanford University, Palo Alto, California, USA (or virtual meeting depending on the COVID-19 situation)

https://sites.google.com/view/aaai-mlps

Key Dates 
Submission: November 20, 2020, 23:59 GMT
Notification: December 18, 2020
Symposium: March 22-24, 2021

With recent advances in scientific data acquisition and high-performance computing, Artificial Intelligence (AI) and Machine Learning (ML) have received significant attention from applied mathematics and physics science community. From successes reported by industry, academia, and research communities, we observe that AI and ML have great potential to leverage scientific domain knowledge to support new scientific discoveries and enhance the development of physical models for complex natural and engineering systems.  

For example, deep learning supports discovery of new materials and high-energy physics from numerous computer simulations and experiments and let us learn low-dimensional manifolds underlying the acquired data in order to represent the system of interest parsimoniously and effectively. ML has offered new insights on adaptive numerical discretization schemes and numerical solvers, which are clearly distinct from traditional mathematical theories. AI also provides a new way of generalizing constitutive physics laws based on big scientific data sets. 

Despite the progress, there are still many open questions. Our current understanding is limited regarding how and why AI/ML work and why they can be predictive. AI has been shown to outperform traditional methods in many cases especially with high-dimensional, inhomogeneous data sets. However, a rigorous understanding of when AI/ML is the right approach is largely lacking: for what class of problems, underlying assumptions, available data sets, and constraints are these new methods best suited? The lack of interpretability in AI-based modeling and related scientific theories makes them insufficient for high-impact, safety-critical applications such as medical diagnoses, national security, and environmental contamination. With transparency and a clear understanding of the data-driven mechanism, desirable properties of AI should be best utilized to extend current methods in physical and engineering modeling. Handling expensive training costs and large memory requirements for ever-increasing scientific data sets becomes also important to guarantee scalable science machine learning. 

This symposium will aim to present the current state of the art and identify opportunities and gaps in AI/ML-based physics science. The symposium will focus on challenges and opportunities for increasing the scale, rigor, robustness, and reliability of physics-informed AI necessary for routine use in science and engineering applications and discuss potential researcher-AI collaborations to significantly advance diverse scientific areas and transform the way science is done.

Topics:
Authors are strongly encouraged to present papers that combine and blend physical knowledge and artificial intelligence/machine learning algorithms. Topics of interest include but are not limited to the following:

1. Artificial intelligence/machine learning framework that can seamlessly synthesize models, governing equations and data
2. Approaches to encode scientific knowledge in machine learning method and architecture
3. Architectural and algorithmic improvements for scalable physics-informed learning
4. Stability and error analysis for physics-informed learning
5. Software development facilitating the inclusion of physics domain knowledge in learning
6. Discovery of physically interpretable laws from data
7. Applications incorporating domain knowledge into machine learning

Submission guideline: 

We solicit extended abstracts, full papers, and poster abstracts on topics related to the above and can include recent or ongoing research, surveys, and business/use cases. Extended abstracts (2 to 4 pages) and full papers (up to 6 pages) will be peer-reviewed. Posters can be proposed by submitting an abstract (1 to 2 pages).

All submissions should follow the AAAI format in the Author Kit, will be handled through EasyChair (https://easychair.org/conferences/?conf=sss21) and the review will be double-blind to ensure academic integrity. Accepted extended abstracts and full papers shall be published in an open access proceedings site. More details can be found in https://easychair.org/cfp/AAAI_MLPS_2021 

Invited Speakers: 
Animashree Anandkumar, Caltech/NVIDIA
Nathan Kutz, University of Washington
More invited speakers are to be announced

Organizing Committee:
Jonghyun Harry Lee, University of Hawai'i at Manoa
Eric F. Darve, Stanford University
Peter K. Kitanidis, Stanford University
Michael W. Mahoney, University of California, Berkeley
Anuj Karpatne, Virginia Tech
Matthew W. Farthing, U.S. Army Engineer Research and Development Center
Tyler Hesser, U.S. Army Engineer Research and Development Center

Program Committee:
Kevin Carlberg, University of Washington
Nathaniel Trask, Sandia National Laboratories
Paris Perdikaris, University of Pennsylvania
Peter Sadowski, University of Hawai'i at Manoa
Hongkyu Yoon, Sandia National Laboratories

**Comment by icsabai** [comment]
Dear Colleagues,

The abstract deadline for Machine Learning in Computational Biology (MLCB) meeting is fast approaching (Sept 30th). Can you please forward the CfP below to relevant mailing lists?

 

Best wishes,

MLCB organizers

https://mlcb.github.io/

__Machine Learning in Computational Biology (MLCB), Nov 23-24, 2020__
We are excited to be holding the 15th Machine Learning in Computational Biology (MLCB) meeting. In its 2020 reincarnation, MLCB will be a two day virtual conference November 23 and 24, 9am-5pm PST. 

 

Since its inception in 2004, and until 2017, MLCB was an official NeurIPS workshop. Given the growth and maturity of the field, MLCB is now an independent meeting.

 

More details can be found on the website: https://mlcb.github.io

Registration: MailScanner has detected definite fraud in the website at "forms.gle". Do not trust this website: https://forms.gle/ZSwPFoLQ3iWBedsj8

Papers from MLCB 2019 can be viewed here: https://mlcb.github.io/mlcb2019_proceedings/

 

Important dates:

Submissions due: Sept 30th, 11:59pm (PST) at https://cmt3.research.microsoft.com/mlcb2020

Decision notification: Oct 30th

Meeting: Nov 23-24th. 

Keynote speakers
Eran Segal (Weizmann Institute of Science)

Aviv Regev (MIT, Broad Institute, Genentech)

Regina Barzilay (MIT)

Marinka Zitnik (Harvard Medical School)

Organizers
Anshul Kundaje - Stanford University (USA)

Su-In Lee - University of Washington (USA)

Sara Mostafavi - University of British Columbia (Canada)

Gerald Quon - UC Davis (USA)

James Zou - Stanford University (USA)

David A Knowles - Columbia University & New York Genome Center (USA)

Meeting description
The field of computational biology has seen dramatic growth over the past few years. A wide range of high-throughput omics and imaging technologies developed in the last decade now enable us to measure parts of a biological system at various resolutions—at the genome, epigenome, transcriptome, and proteome levels. These diverse technologies are now being used to study questions relevant to basic biology and human health. Fully realizing the scientific and clinical potential of these data requires developing novel supervised and unsupervised learning methods that are scalable, can accommodate heterogeneity, are robust to systematic noise and confounding factors, and provide mechanistic insights.

 

The goals of the MLCB meeting are to i) present emerging problems and innovative machine learning techniques in computational biology, and ii) generate discussion on how to best model the intricacies of biological data and synthesize and interpret results in light of the current work in the field.

Submission instructions
We will accept submission in two tracks: 

1.    Research on novel learning approaches in computational biology. We encourage contributions describing either progress on new problems or work on established problems using methods that are substantially different from established alternatives.

2.    Perspective on important but controversial problems. Examples include (but are not limited to) different ways of conceptualizing the supposed dichotomy between interpretation and accuracy, over-interpretation from visual representations of high dimensional data, lack of statistical rigour in inferring conclusions from data and models. 

 

Researchers interested in contributing should upload a short paper (4 pages) in PDF format to the MLCB submission web site:

https://cmt3.research.microsoft.com/mlcb2020

by Sept 30th, 11:59 pm (PST). Both tracks will undergo rigorous peer review as in previous years. 

 

No special style is required. Authors may use the NeurIPS style file, but are also free to use other styles as long as they use standard font size (10 or 11pt) and margins (1 in). 

 

Papers need not be anonymized (but you can choose to anonymize your submission if you wish). 

 

All submissions to the Research track will be peer reviewed and will be evaluated on the basis of their technical content.  A strong submission to the meeting typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online at http://raetschlab.org:10080/nipscompbio/previous and https://mlcb.github.io/mlcb2019_proceedings/. 

 

The meeting allows submissions of papers that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at the time of submission (e.g. as a footnote on the first page).

**Comment by icsabai** [comment]
Ha nem jött volna más csatornán, a GTC-re lehet ingyen regisztrálni diákoknak és oktatóknak:

I’d like to invite you to attend NVIDIA’s GPU Technology Conference (GTC) Oct. 5-9. At this premiere online technical event, you’ll learn about the latest breakthroughs in AI, HPC, graphics, data science and more. Best of all, registration is free for educators, students, and developers in the higher education and research ecosystem; just sign up using your university email.

GTC will run continuously, across all time zones. Connect with fellow developers, faculty, researchers, students, and research IT and learn how to speed up time to insights for your research.

Discover the infrastructure needed to build an AI-powered university in this expert-led panel discussion featuring NVIDIA, the University of Florida, and the Joint Artificial Intelligence Center (JAIC).

In another not-to-miss event, see how startups are pushing the boundaries of IVA and advanced speech processing using AI. On Tuesday, October 6 at 12PM PDT, NVIDIA CEO Jensen Huang will be joined by CrowdAI CEO Devaki Raj and BabbleLabs CEO Chris Rowen in the Fireside Chat. Learn how NVIDIA supports thousands of startups’ developers through its acceleration program, NVIDIA Inception.

Check out our session scheduler or all available sessions and training within Higher Ed and Research. Here’s a snapshot of top sessions and events you might enjoy:
Realizing the Vision of an AI University
Perception and Reinforcement Learning: From Research to Practice
Implementing GPUs for AI-Driven Research and Quantum Mechanics Calculations
A Deep Dive into the Latest HPC Softwar
Check out the inaugural AI Art Gallery or the public sector special event, Building an AI Nation
Remember, registration is free for all educators and students with a university email. If the registration does not recognize your university email address, please click here to create an account and then send an email to GTC_US@nvidia.com . with proof of your academic status. They will then apply the complimentary pricing to your account.

Hope to see you at the show,

Cheryl Martin
Director, AI in Higher Education, NVIDIA
2788 San Tomas Expressway Santa Clara, CA 95051
www.nvidia.com •  opt-out

**Comment by abiricz** [comment]
Ez tök jó, szívesen regisztálok és belehallgatok, meg is csináltam már igazából, de nem ismeri fel az ELTE-t a felület. Így e-mailt kell nekik írni, viszont valahogy szerezni kell akkor angolul egy igazolást, hogy itt tanulunk az ELTE-n. Talán Ibolya tudna ebben viszonylag gyorsan segíteni, a qter-en hosszú sor után lehet csak bejelentkezni jogviszony igazolásra (kötelező az időpontfoglalás).

**Comment by icsabai** [comment]
Már nem tudom mi a jó módszer erre. Én részletesen el szoktam magyarázni !! egyetemi címről küldött !! emailben, hogy ez egy nem amerikai egyetem, nincs bejáratott módja az igazolásnak, a Neptun csak magyarul van és nem publikus. Lemásolhatod nekik a diákigazolványodat, megadhatod a csabaibio oldalt http://csabaibio.elte.hu/team.html Illetve itt https://physics.elte.hu/KRFT_machinelearning is fel vagy sorolva. Vagy ebben a hírben https://www.elte.hu/content/elte-s-hallgatok-8-ora-alatt-allitottak-ossze-egy-gepilatas-programot.t.19181  (csak van le tudják angolra fordítani ...)

**Comment by icsabai** [comment]
Barnaföldi Gergőék szervezik a köv. GPU napot:

_Jövő héten kedden/szerdán lenne a GPU nap. Szóban már kérdeztem, de
most írásban is rákérdezek, hogy te vagy a diákjaid tartanátok--e
előadást? Ha igen akkor a gpuday.com-on regisztráljatok és küldjétek el
nekem a címet kivonatot (angolul)._

https://gpuday.com/gpu-day-2020

**Comment by icsabai** [comment]
MI Koalíció rendezvény:

Tisztelt Munkacsoport tagok!

 

Hosszú szünet telt el a legutóbbi összejövetelünk óta, ugyanakkor sok minden történt a mesterséges intelligencia területén.

Számunkra a legfontosabb hazai mérföldkövek: a Mesterséges Intelligencia Nemzeti Stratégia elfogadása és bejelentése, a Mesterséges Intelligencia Nemzeti Laboratórium (MILAB) elindulása és a Nemzeti Adatvagyon Ügynökség (NAVÜ) elindítása. Valamint, a Mesterséges Intelligencia Koalíció munkájának újraindítása. Az elkészült Stratégia olyan vállalásokat, célokat tartalmaz, amelyek megvalósítása sok munkát ad majd a piac összes szereplőjének.

 

Erős felütéssel szeretnénk kezdeni a munkát és ehhez egy nagyon érdekes témakörben, a mesterséges intelligenciához kapcsolódó biztonsággal foglalkozó webinárra invitálom most Önöket. Előadóink a MILAB részéről a Szegedi Tudományegyetem és a BME CrySyS Lab kutatói, valamint a Cydrill Information Security munkatársai. Az előadások során és után lehetőség lesz kérdéseket feltenni, amit az előadók igyekeznek megválaszolni. Illetve lesz lehetőség a vélemények kicserélésére, ismerkedésre.

 

Időpont:

·         2020. október 20. 14-től 17 óráig

Helyszín:

·         ZOOM, online tér

__Előadások:__

·         _Mesterséges neuronhálók érzékenységi problémája: alapfogalmak és néhány eredmény_ – __Dr. Jelasity Márk__, Szegedi Tudományegyetem

·         _Certified Robustness to Adversarial Examples_ – __Dr. Ács Gergely__, BME CrySyS Lab

·         _100 ways your machine learning systems are vulnerable_ –__ Kiss Balázs__, Cydrill

 

A rendezvényre meghívjuk az Alkalmazások munkacsoport tagjait és a MILAB kutatói közösségét is.

 

Hamarosan küldöm a rendezvény linkjét! Szeretettel várunk mindenkit!

**Comment by icsabai** [comment]
CSFK CSI SZEMINÁRIUM / KONKOLY OBSERVATORY SEMINAR
 
 Speaker: Tamás Szklenár (Konkoly Observatory)
 
 Title: __The opportunities of Machine Learning methods in astronomy__
 
 Date: 29 October, 2020 (Thursday), 14:00
 
 Note that because of the COVID-19 pandemic, the Konkoly seminars will be held online until further notice. Take the opportunity and join us from anywhere, using the Zoom videoconferencing system (see the connection details below). 
 
 Everyone is welcome!
 
 Sándor Frey, Krisztián Vida
 organizers
 konkoly.hu/szeminar/szeminarium.shtml
 
 CSI zoom is inviting you to a scheduled Zoom meeting.Topic: Konkoly seminarTime: Oct 29, 2020 02:00 PM Budapest
Join Zoom Meeting:
http://zoom.us/j/97108889740?pwd=Y1ZOSmIxNjYxckNZODJocjJ1dUZ6QT09
Meeting ID: 971 0888 9740
Password: 070804

**Comment by icsabai** [comment]
Aki szeretné meghallgatni a Mesterséges Intelligencia Nemzeti Labor tájékoztató konferenciáját jövő szerdán, regisztráljon az alábbiak szerint:


Tárgy: MEGHÍVÓ_MILAB BEMUTATÓ_2021.02.17.

 

Tisztelt Partnerünk!

 

A Nemzeti Laboratórium rendszere azzal a céllal jött létre, hogy a felfedező és kísérleti megközelítésű kutatásoknak új, nemzetközi dimenzióját nyitó, együttműködésen alapuló, intézményesülő, dinamikus színteret biztosítson a kutatási eredmények társadalmi, gazdasági, környezeti hasznosítására.

 

Hagyományteremtő céllal, az egyes Laboratóriumok munkájának megismerése, szinergiák keresése és együttműködések kialakítása érdekében elindításra kerül egy belső eseménysorozat, amelynek keretében először a Mesterséges Intelligencia Nemzeti Laboratórium (MILAB) mutatkozik be a hálózat tagjainak.

 

Ezúton tisztelettel meghívjuk Önt és a Nemzeti Laboratórium Programban részt vevő kollégáit a 2020. február 17. napon 14:00 órai kezdettel megrendezésre kerülő MILAB bemutatkozó online konferenciára.

 

Az esemény előzetes regisztrációhoz kötött, ezért kérjük, részvételi szándékát a milab@sztaki.hu e-mail címen jelezze.

 

A regisztrációt követően a szervezők továbbítani fogják a bejelentkezéshez szükséges információkat, linket.

 

Részvételére feltétlenül számítunk!

[PROGRAMTERV_MILAB_bemutatkozó_konferencia.pdf](https://github.com/csabaiBio/elte_ml_journal_club/files/5968385/PROGRAMTERV_MILAB_bemutatkozo_konferencia.pdf)

**Comment by icsabai** [comment]
Sziasztok, ahogy ma említettem, lesz jövő héten a [Reinforce Conference](https://reinforceconf.com/). Sajnos nem ingyenes kupont adtak, hanem 30% OFF.  Ez a kupon:

![IstvanSentMe30](https://user-images.githubusercontent.com/18164480/109201349-08254680-77a2-11eb-85c9-9811ecfdce98.png)

**Comment by qbeer** [comment]
Nvidia GTC ingyen regisztráció és részvétel: https://gtc21.event.nvidia.com/

**Comment by icsabai** [comment]
Eastern European Machine Learning Summer School
7-15 July 2021, Virtual Budapest Hungary

Deep Learning and Reinforcement Learning

https://www.eeml.eu/home

**Comment by icsabai** [comment]
Kedves Kollégák!

A Rényi Intézet Deep Learning szemináriumának következő előadását __Olár Alex__ (ELTE TTK Fizikai Intézet) tartja június 2-án szerdán 16:00-kor, zoom link és absztrakt alább.

https://zoom.us/j/92806934772?pwd=L1Q0MmRuSURpSmU3cVo2Wit6L3RTZz09

DNS szekvenálás gépi tanulással

A Humán Genom Projekt keretében 2003-ban sikeresen leolvasták és összeillesztették a teljes emberi genom szekvenciát. Az elmúlt közel 20 évben a technológiai fejlődésnek köszönhetően a genom szekvenálás ára exponenciálisan csökkent, így egyre nagyobb mértékben elerhető és tanulmányozható ilyen típusú adat. Ma már valós-idejű DNS szekvenálásra is lehetőség van (Oxford Nanopore Technologies Ltd.), ami lehetővé teszi a gyors vizsgálatokat és minta szelekciót. Ezen feltételek mellett nem meglepő, hogy rengeteg területen kezdtek el gépi tanulást alkalmazni, metilációs vizsgálatokra, transzkripciós faktor jóslására, base-callingra, amely során közvetve olvassák le a bázis szekvenciát. A gépi tanulás felfutásával, kezdetben rekurrens hálókat, konvolúciós LSTM hálókat és ma már Transformer alapú modelleket is használnak a genomika különböző területein.

Az előadás során néhány témába vágó cikket fogok bemutatni, amelyek elsősorban mai, természetes nyelv feldolgozási technikákat alkamaznak a genomikára, ilyen az idén megjelent DNABERT [1], amely gyakorlatilag DNS-nyelvfeldolgozást csinál és a fine-tuning lépések után a figyelem mátrixból próbál kiolvasni releváns információt. A valós idejű szekvenálást egyenlőre rosszabb pontosságú, mint a korábban kifejlesztett technikák, de kifejlesztése óta (2014) nagy lendülettel fejlesztik a vizsgálati eszközöket, valamint a feldolgozó algoritmusokat is. Egy áttekintő cikk [2] keretében felvázolom a jelenlegi algoritmusokat [3] és bemutatok néhány újkeletűt is [4]. Ezen módszerek lényege, hogy egy feszültség jelet alakítsanak át az annak megfelelő bázis-szekvenciává, amelyben hasznosak lehetnek a mai text-to-speech technikák.

[1] Ji, Yanrong, et al. "DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome." bioRxiv (2020).
[2] Rang, Franka J., Wigard P. Kloosterman, and Jeroen de Ridder. "From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy." Genome biology 19.1 (2018): 1-11.
[3] Teng, Haotian, et al. "Chiron: translating nanopore raw signal directly into nucleotide sequence using deep learning." GigaScience 7.5 (2018): giy037.
[4] Lv, Xuan, et al. "An End-to-end Oxford Nanopore Basecaller Using Convolution-augmented Transformer." 2020 IEEE International Conference on Bioinformatics and Biomedicine (BIBM). IEEE, 2020.

**Comment by icsabai** [comment]
An upcoming talk on quantum machine learning:


The BME Exotic Quantum Phases seminar continues on Tuesday at 14:15 with this talk:

    Vincent Elfving (CTO, Qu & Co, Amsterdam - https://quandco.com/)
    Solving Nonlinear Differential Equations with Differentiable Quantum Circuits

online in Teams
Time: Sep 28 Tuesday, 14:15
Abstract & Reference: https://journals.aps.org/pra/abstract/10.1103/PhysRevA.103.052416

About the speaker: Vincent is a quantum-physicist with a doctorate from the University of Copenhagen where his research focused on quantum-computational chemistry and theoretical quantum-optics at the Niels Bohr Institute under supervision of professor Anders Sørensen. Previously he completed his Masters at the University of Tokyo and Bachelors at Delft University of Technology. Vincent also worked as a researcher at Google, in the quantum-hardware team of John Martinis, where he focused on implementing quantum-computational chemistry simulations and quantum optimization strategies on current-day quantum-processors with three different superconducting qubit architectures. In his role as CTO of Qu & Co, Vincent leads the company's quantum-algorithm development activities and our implementations of such algorithms on the quantum-processors of hardware partners.

Teams meeting link:
https://teams.microsoft.com/l/meetup-join/19%3a98c537f3ff3a433983cc04579e4603af%40thread.tacv2/1632480203968?context=%7b%22Tid%22%3a%226a3548ab-7570-4271-91a8-58da00697029%22%2c%22Oid%22%3a%22f8b27e90-dbca-4bb5-813c-d6415577ec35%22%7d

**Comment by icsabai** [comment]
NVIDIA is one of the major suppliers of GPU cards and developer of CUDA that is one of the workhorses of machine learning frameworks.
Their conference [GTC](https://www.nvidia.com/gtc/) starts today. With free registration you can follow it online. As always we expect several interesting talks and announcements. If someone finds something really interesting, you are welcome to give a summary presentation at one of the next seminars!
